liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/CreateServerMojo.java

```diff
@@ -37,6 +41,40 @@
      */
     private String template;
     
+    /**
+     * Packages to install. One of "all", "dependencies" or "project".
+     * 
+     * @parameter property="installAppPackages" default-value="dependencies"
+     * @readonly
+     */
+    private String installAppPackages = "dependencies";
+    
```

Comment:
I was surprised to see the properties show up again from InstallAppMojo and InstallAppMojoSupport, but looking at the inheritance I don't see a good place to combine these in one place.  If we can't put in one common class, we will have to be careful that the default-value stays in sync.  Specifically when we remove the default-value for dropins on the appsDirectory.  Would be nice for each mojo to supply its own relevant config.